tests/detectors/test_face_detector.py

```diff
@@ -11,92 +11,130 @@
 from os.path import abspath
 
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)